io.github.tareksander.story_reader.yaml

```diff
@@ -0,0 +1,40 @@
+id: io.github.tareksander.story_reader
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+command: story_reader
+sdk: org.freedesktop.Sdk
+finish-args: [
+  "--share=network",
+  "--socket=pulseaudio",
+  "--socket=fallback-x11",
+  "--socket=wayland",
+  "--share=ipc",
+  "--device=dri"
+]
+cleanup: [
+  "/lib"
+]
+modules:
+  - name: app
```

Comment:
please use a descriptive name here.

Reply:
Better now? As you may have noticed with the app name, I'm quite bad at naming things.